src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { FC } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink/SearchLink';
+
+interface PeopleFiltersPerops {
+  query: string
+  sex: string | null
+  centuries: string[]
+}
+
+export const PeopleFilters: FC<PeopleFiltersPerops> = ({
+  query,
+  sex,
+  centuries,
+}) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const centuriesArray: string[] = ['16', '17', '18', '19', '20'];
+
```

Comment:
You can consider creating a constants file to store the 'centuriesArray' and import it whenever needed, rather than hardcoding it here.